app/src/organisms/Devices/PipetteCard/__tests__/PipetteSettingsSlideout.test.tsx

```diff
@@ -1,28 +1,33 @@
 import * as React from 'react'
 import { resetAllWhenMocks } from 'jest-when'
+import omit from 'lodash/omit'
 import { renderWithProviders } from '@opentrons/components'
-import { fireEvent } from '@testing-library/react'
+import { pipetteSettingsResponseFixture } from '@opentrons/api-client/src/pipettes'
+import { act } from 'react-dom/test-utils'
 import { i18n } from '../../../../i18n'
 import * as RobotApi from '../../../../redux/robot-api'
-import { ConfigurePipette } from '../../../ConfigurePipette'
+import { getAttachedPipetteSettingsFieldsById } from '../../../../redux/pipettes'
+import { getConfig } from '../../../../redux/config'
 import { PipetteSettingsSlideout } from '../PipetteSettingsSlideout'
 
 import { mockLeftSpecs } from '../../../../redux/pipettes/__fixtures__'
 
 import type { DispatchApiRequestType } from '../../../../redux/robot-api'
 
-jest.mock('../../../ConfigurePipette')
 jest.mock('../../../../redux/robot-api')
+jest.mock('../../../../redux/config')
+jest.mock('../../../../redux/pipettes')
 
-const mockConfigurePipette = ConfigurePipette as jest.MockedFunction<
-  typeof ConfigurePipette
->
+const mockGetConfig = getConfig as jest.MockedFunction<typeof getConfig>
 const mockUseDispatchApiRequest = RobotApi.useDispatchApiRequest as jest.MockedFunction<
   typeof RobotApi.useDispatchApiRequest
 >
 const mockGetRequestById = RobotApi.getRequestById as jest.MockedFunction<
   typeof RobotApi.getRequestById
 >
+const mockGetAttachedPipetteSettingsFieldById = getAttachedPipetteSettingsFieldsById as jest.MockedFunction<
```

Comment:
Slight mismatch in mock name vs dependency name (missing an `s`)

```suggestion
const mockGetAttachedPipetteSettingsFieldsById = getAttachedPipetteSettingsFieldsById as jest.MockedFunction<
```